Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -325,7 +372,7 @@ WarpXOpenPMDPlot::DumpToFile (WarpXParticleContainer* pc,
          const auto& aos = pti.GetArrayOfStructs();  // size =  numParticlesOnTile
          {
            // Save positions
-           std::vector<std::string> axisNames={"x", "y", "z"};
+           auto const positionComponents = detail::getParticlePositionComponents();
```

Comment:
Thanks for this PR @ax3l ! This is super organized. I will try to think about how to transport this into the labels of the components in the input too (I mean in the docs).